sublime-package.json

```diff
@@ -84,81 +84,80 @@
-                        "typescriptreact": {
-                          "type": "boolean",
-                          "default": false
+                          "default": false,
+                          "type": "boolean"
                         },
                         "javascriptreact": {
-                          "type": "boolean",
-                          "default": false
+                          "default": false,
+                          "type": "boolean"
                         },
-                        "css": {
-                          "type": "boolean",
-                          "default": false
+                        "json": {
+                          "default": false,
+                          "type": "boolean"
+                        },
+                        "jsonc": {
+                          "default": false,
+                          "type": "boolean"
+                        },
+                        "less": {
+                          "default": false,
+                          "type": "boolean"
                         },
                         "scss": {
-                          "type": "boolean",
-                          "default": false
+                          "default": false,
+                          "type": "boolean"
                         },
-                        "json": {
-                          "type": "boolean",
-                          "default": false
+                        "typescript": {
+                          "default": false,
+                          "type": "boolean"
                         },
-                        "jsonc": {
-                          "type": "boolean",
-                          "default": false
+                        "typescriptreact": {
+                          "default": false,
+                          "type": "boolean"
                         }
-                      }
+                      },
+                      "type": "object"
                     },
-                    "volar.vueserver.textDocumentSync": {
-                      "type": "string",
-                      "default": "incremental",
-                      "enum": [
-                        "incremental",
-                        "full",
-                        "none"
-                      ],
-                      "enumDescriptions": [
-                        "Documents are synced by sending the full content on open. After that only incremental updates to the document are send.",
-                        "Documents are synced by always sending the full content of the document.",
-                        "Documents should not be synced at all."
-                      ],
-                      "description": "Defines how the host (editor) should sync document changes to the language server. SFC incremental parser only working when config \"incremental\"."
+                    "volar.updateImportsOnFileMove.enabled": {
```

Comment:
this setting doesn't work in ST,
does it work for you.

I renamed the file in the file manager,
and Volar did not update the import.

Reply:
It might not work, haven't checked. I just saw that there is no special logic for it in the extension but maybe it also depends on some LSP feature that we don't implement?

Reply:
Found the relevant code `connection.workspace.onWillRenameFiles`:

https://github.com/johnsoncodehk/volar/commit/fd4d125fb8ed46ac962b65f938971ebc26bc223c#diff-ed88d2723f7453a56638acb05d1beaf4af8d3d8658689de17e8f86c16b8aeb29R283

Reply:
I would guess that we will remove this setting for now.